Dockerfile-platform-mock

```diff
@@ -11,7 +11,7 @@ ADD scl-enable.sh $APPBASEDIR/
 
 RUN $APPBASEDIR/scl-enable.sh pip install --upgrade pip
 
-RUN $APPBASEDIR/scl-enable.sh pip install tornado "aiokafka==0.5.0" insights-core
+RUN $APPBASEDIR/scl-enable.sh pip install tornado "aiokafka==0.5.0" insights-core "watchtower==0.5.5"
```

Comment:
This is forgotten by me to add requirements.txt file alsof for platform-mock, so it would be good to add it. In the other dockerfiles it`s already used.

Reply:
Moreover using of quotation marks ("aiokafka==0.5.0") is useless, isn't it?

Reply:
platform mock isn't deployed anywhere, feel free to create requirements.txt for it but it isn't priority

not sure if quotation marks are needed here, anyway they are harmless, they would be definitely needed if there is for example "watchtower>=0.5.5"

Reply:
That's true, it's needed when using `>=` but not for `==`.